Open Judge System/Web/OJS.Web/Areas/Administration/Controllers/KendoRemoteDataController.cs

```diff
@@ -98,5 +125,20 @@ public JsonResult GetCascadeProblemGroupsFromContest(int contestId)
 
             return this.Json(result, JsonRequestBehavior.AllowGet);
         }
+
+        [AjaxOnly]
+        public JsonResult GetCascadeProblemsFromContest(int contestId)
+        {
+            var problems = this.problemsData.GetAllByContest(contestId);
+
+            var result = problems
+                .Select(p => new DropdownViewModel
```

Comment:
I personally like this formatting, but I think `Select` should be on the above like. Similarly to `ZippedTestsParser`, line 84.